_includes/package_body.html

```diff
@@ -429,3 +443,70 @@ <h3 class="panel-title">Plugins</h3>
 </div>
 
 <script src={{ "/js/package_body_tabs.js" | prepend: site.baseurl }}></script>
+
+<script type="text/javascript" src={{"/js/jquery.jfeed.pack.js" | prepend: site.baseurl }}></script>
+
+<script type="text/javascript">
+jQuery.browser = {};
+(function () {
+    jQuery.browser.msie = false;
+    jQuery.browser.version = 0;
+    if (navigator.userAgent.match(/MSIE ([0-9]+)\./)) {
+        jQuery.browser.msie = true;
+        jQuery.browser.version = RegExp.$1;
+    }
+})();
+jQuery(function() {
+    jQuery.getFeed({
+        url: 'https://answers.ros.org/feeds/rss/?tags={{package.name}}',
+        success: function(feed) {
+            var html = '';
+
+            if (feed.items.length == 0) {
+              html = 'No questions yet, you can ask one <a href="https://answers.ros.org/questions/ask/?tags={{package.name}},{{distro}}">here</a>.'
+              jQuery('#{{distro}}-answers-result').append(html);
+              jQuery('#{{distro}}-answers-count').replaceWith("0")
+              return
+            }
+            html += '<p>Browse '
+            + '<a href="https://answers.ros.org/questions/scope:all/sort:activity-desc/tags:{{package.name}}/page:1/">recent questions</a>'
+            + ' or see the '
+            + '<a href="https://answers.ros.org/questions/scope:all/sort:votes-desc/tags:{{package.name}}/page:1/">higest voted</a>'
+            + '</p>'
+
+            html += '<div class="question_list">'
+            + '<h4>Recent questions preview:</h4>'
+            + '<ul>'
+            for(var i = 0; i < feed.items.length && i < 50; i++) {
+
+                var item = feed.items[i];
+
+                html += '<li>'
```

Comment:
@tfoote I wonder if it'd make sense to use a templating library (e.g. [mustache](http://mustache.github.io/)). That would simplify building more complex and better looking listings, like [list groups](https://getbootstrap.com/docs/4.0/components/list-group/#custom-content) or in wells.